docs/adr/0007-apply-dio-cache-interceptor.md

```diff
@@ -0,0 +1,51 @@
+# 7. Apply DioCacheInterceptor for caching API
```

Comment:
![Screenshot 2023-09-19 at 14 06 47](https://github.com/linagora/twake-on-matrix/assets/12546908/a096daf6-81fb-4118-baf2-fcb20835e931)
It should be `#8` for now :D 
can you also update `#6` to `#7`

Reply:
Done